### ISSUE TITLE
Problem with setting Authorization header

### ISSUE THREAD

**Issue by antonkomarev** [nicklandgrebe/active-resource.js#37]
This code works:
```js
let jsonApi = createResourceLibrary('http://127.0.0.1:8000', {
  headers: {
    Authorization: `${tokenType} ${accessToken}`,
  },
});
```

This one doesn't send headers when doing request:
```js
let jsonApi = createResourceLibrary('http://127.0.0.1:8000');
jsonApi.headers = {
    Authorization: `${tokenType} ${accessToken}`,
};
```

And I've noticed that first example adds extra header:
```js
'Content-Type': 'application/vnd.api+json',
```
While second one doesn't do it. And it seems that both examples doesn't use this header in requests.

**Comment by Chilinot** [comment]
I have the same issue. I cant get the requests to authenticate against the backend because i define the headers after the createResourceLibrary call.

**Comment by nicklandgrebe** [comment]
Sorry for the delay. I have uploaded the fix for this to master, and will update you when it is released as `1.0.0-beta.4`

**Comment by fconforti** [comment]
@nicklandgrebe any updates for the 1.0.0-beta.4?

Thanks,
F.
